standards/ruby.md

```diff
@@ -0,0 +1,46 @@
+# <a href="#ruby"></a>Ruby Best Practices
```

Comment:
:+1 to this entire file, while there may be exceptions, i do believe a lot of these tools significantly aid quality development.